### DISCUSSION TITLE
Sorting repeated command option and allowing full screen.

### DISCUSSION THREAD

**Discussion by simurgh9** [rxhanson/Rectangle#1364] [Ideas]
The repeated commands drop down in the settings tab has an option "cycle 1/2, 2/3, and 1/3 on half actions."  This comes in really handy for me.

Could we add an option for say, "cycle 1/3, 1/2, 2/3, and 1 on half actions?" Sometimes I sort of go through different sizes of a window to feel for the right size.  It'll be handy if I could iterate over all fractions of screen coverage under repeated keypresses.

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have plans to add something like this into Rectangle in the near future, but if anyone wanted to add it in as a Terminal command configurable option (not in the UI), I see no reason to turn down a good pull request for it. 

As an alternative, it is possible to build your own shortcut in [Rectangle Pro](https://rectangleapp.com/pro) (paid). Here's a quick example for doing it with left half:
![image](https://github.com/rxhanson/Rectangle/assets/13651296/a46e103a-c044-4aaf-a07b-4d9b54fe825e)
